Frontend/CO.CDP.OrganisationApp/Pages/Supplier/ConnectedEntity/ConnectedEntityCheckAnswersIndividualOrTrust.cshtml

```diff
@@ -41,7 +42,7 @@
                             <p class="govuk-body">@details.LastName</p>
                             @if (details.DateOfBirth.HasValue)
                             {
-                                <p class="govuk-body">DoB: @details.DateOfBirth?.ToString("dd/MM/yyyy")</p>
+                                <p class="govuk-body">DoB: @details.DateOfBirth?.ToString("dd MMMM yyyy")</p>
```

Comment:
are we not able to use `ToFormattedString` here also?

Reply:
Done